go/libkb/identity.go

```diff
@@ -19,8 +19,8 @@ type Identity struct {
 
 func ParseIdentity(s string) (*Identity, error) {
 	rxx := regexp.MustCompile("" +
-		`^([^(<]*?)` + // The beginning name of the user (no comment or key)
-		`(?:\s*\((.*?)\))?` + // The optional comment
+		`^"?([^(<]*?)"?` + // The beginning name of the user (no comment or key)
```

Comment:
Yeah, the idea is to strip out the quotation marks.
Since they can span across the name and comment, it doesn't make sense to leave them in the name.
Examples:
`"Barb Akew"` -> `Barb Akew`
`"Barb Akew (comment)"` -> `Barb Akew`, `comment`

Inner quotes are not removed so a comment like `(the "new" key)` would keep its inner quotes.